src/utils/MultiBuffer.cpp

```diff
@@ -851,17 +855,17 @@ void MultiBuffer::unpack_data (int slice, MultiBeam& beams, MultiLaser& laser, i
             }
         }
     }
-    if (m_use_laser) {
+    if (laser.UseLaser(slice)) {
```

Comment:
That's the skip if current slice has no laser right?

Reply:
Yes